controller/konnect/konnectextension_controller.go

```diff
@@ -26,31 +27,37 @@ import (
 	"github.com/kong/gateway-operator/controller/pkg/patch"
 	"github.com/kong/gateway-operator/internal/utils/index"
 	"github.com/kong/gateway-operator/pkg/consts"
+	konnectresource "github.com/kong/gateway-operator/pkg/utils/konnect/resources"
 	k8sutils "github.com/kong/gateway-operator/pkg/utils/kubernetes"
 
-	commonv1alpha1 "github.com/kong/kubernetes-configuration/api/common/v1alpha1"
+	configurationv1alpha1 "github.com/kong/kubernetes-configuration/api/configuration/v1alpha1"
 	operatorv1alpha1 "github.com/kong/kubernetes-configuration/api/gateway-operator/v1alpha1"
 	operatorv1beta1 "github.com/kong/kubernetes-configuration/api/gateway-operator/v1beta1"
 	konnectv1alpha1 "github.com/kong/kubernetes-configuration/api/konnect/v1alpha1"
 )
 
+const certificateIDsAnnotationKey = "konnect.konghq.com/certificate-ids"
```

Comment:
- Can we add a godoc for this to describe where/when this annotation is used?
- BTW: this has the same value as `DataPlaneCertificateIDLabelKey`, can we reuse the const (add a separate one but use `=` to assign the value)?

Reply:
Good catch, these two consts were actually duplicates. I got rid of this one in favor of the already documented in the consts package.